core/src/elements/ons-list-item.js

```diff
@@ -232,16 +236,9 @@ export default class ListItemElement extends BaseElement {
     }
   }
 
-  _tapCheck() {
-    return [
-      e => e.hasAttribute('prevent-tap') || /^ons-(?!col$|row$)/i.test(e.tagName),
-      e => e === this
-    ];
-  }
-
   _onTouch(e) {
-    if (this.tapped || !this._tapCheck()[1](e.target) &&
-      (this._tapCheck()[0](e.target) || util.findParent(e.target, ...this._tapCheck))
+    if (this.tapped || this !== e.target &&
+      (this._shouldIgnoreTap(e.target) || util.findParent(e.target, this._shouldIgnoreTap, p => p === this))
```

Comment:
It looks like adding `prevent-tap` on `ons-list-item` itself will have no effect, only its children. Is this correct?

On another note, I found this slightly confusing to read. Maybe add one extra set of brackets so it reads like this:

```
this.tapped || 
(this !== e.target && (this._shouldIgnoreTap(e.target) || util.findParent(e.target, this._shouldIgnoreTap, p => p === this))
```

Reply:
That's correct, instead of `prevent-tap` they should just remove `tappable` from `ons-list-item`. I'll try to make that more readable, thanks!